packages/api/src/praise/routes.ts

```diff
@@ -10,11 +10,13 @@ praiseRouter.getAsync('/:id', controller.single);
 praiseRouter.patchAsync(
   '/:id/quantify',
   authMiddleware(UserRole.QUANTIFIER),
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
   controller.quantify
 );
 praiseRouter.patchAsync(
   '/quantify',
   authMiddleware(UserRole.QUANTIFIER),
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Hm. This is should probably not be here, but I see an error is raised otherwise. Let's leave it there since the issue will be no longer after migration to api2.